.github/workflows/smoke-test-daily.yml

```diff
@@ -163,7 +164,7 @@ jobs:
                   A_PW: ${{ secrets.SMOKE_TEST_PERF_ADMIN_PASSWORD }}
                   C_USER: ${{ secrets.SMOKE_TEST_PERF_ADMIN_USER }}
                   C_PW: ${{ secrets.SMOKE_TEST_PERF_ADMIN_PASSWORD }}
-                  P_ID: 274
+                  P_ID: 22733
```

Comment:
The old `P_ID` in the daily perf site no longer exists, and it's causing the daily k6 tests in `trunk` to fail. I'm now updating the `P_ID` with its new value here.